packages/landing/src/config.map.ts

```diff
@@ -229,8 +229,8 @@ export interface LayerInfo {
   /** Layer category */
   category?: string;
   /* Bounding box */
-  upperLeft: [number, number];
-  lowerRight: [number, number];
+  upperLeft?: [number, number];
+  lowerRight?: [number, number];
```

Comment:
This was incorrectly typed as the aerial and topographic layers did not have a upperLeft/LowerRight